basic-javascript/project-1.js

```diff
@@ -31,22 +58,44 @@ const greaterThanFifty = num => {
 };
 
 const add = (x, y) => {
-  return x + y;
+  if (typeof(x) !== 'number' || typeof(y) !== 'number'){
+    return 'values must both be numbers';
+  }
+  const a = x || 0;
+  const b = y || 0;
+  return a + b;
 };
 
 const subtract = (x, y) => {
-  return x - y;
+  if (typeof(x) !== 'number' || typeof(y) !== 'number'){
+    return 'values must both be numbers';
+  }
+  const a = x || 0;
+  const b = y || 0;
+  return a - b;
 };
 
```

Comment:
Interesting use of the || here!

Reply:
thanks, I borrowed this from the example we saw in lecture.